### ISSUE TITLE
Sitewide: [screen reader] Directions to facility link, alt text renders "Directionsto facility."

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#20432]
14321
## Instructions

1. Click or press the gear icon next to the "Labels" heading on the right. Search for your team in the labels list. If your team is not listed in the Labels menu, please leave a comment after you have created this issue. The Product Support team will create a new label for your team.
2. If this issue is being opened by a VFS team, please add the `vsp-product support` label.
3. **Issue tickets should be tested when you have created a pull request.** This gives the 508, or quality assurance, person time to review your code before it is merged into the `master` branch. Once code is merged into `master` it will be deployed to Staging. Reviewing and correcting issues becomes more difficult at this point.
4. Text inside `<-- comment -->` blocks will not appear in your issue ticket. These comments offer guidance on what information you should provide for each section.
5. Leave a link here to the 508 defect severity level: 

    - [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 
Space should be inserted within the alt text.
## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact
Angela Fowler

**VFS Point of Contact:** _First name only. Delete if not applicable._
Angela
## User Story or Problem Statement
The omission of a space within the alt-text causes the link description to be somewhat garbled potentially confusing screen reader users.

## Details
Sitewide: [screen reader] Directions to facility link, alt text renders "Directionsto facility."

## Acceptance Criteria
All facilities located in search results have a link by which the user can retrieve directions. Each of these links's alt text says "directions to facility." In each case, there is no space between "directions" and "to."

**Comment by mmiddaugh** [comment]
@AngelaFowler82 I have moved information from this issue into an issue titled "Update accessibility for "Get Directions" link on Facility Locator" (#25759). That issue will handle more comprehensive improvements to the "Get Directions" link on Facility Locator. 
Closing this
